backend/src/main/java/com/bakdata/conquery/models/query/concept/ArrayQuery.java

```diff
@@ -0,0 +1,62 @@
+package com.bakdata.conquery.models.query.concept;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import com.bakdata.conquery.ConqueryConstants;
+import com.bakdata.conquery.io.cps.CPSType;
+import com.bakdata.conquery.models.identifiable.ids.specific.ManagedExecutionId;
+import com.bakdata.conquery.models.query.IQuery;
+import com.bakdata.conquery.models.query.QueryPlanContext;
+import com.bakdata.conquery.models.query.QueryResolveContext;
+import com.bakdata.conquery.models.query.queryplan.ArrayQueryPlan;
+import com.bakdata.conquery.models.query.resultinfo.ResultInfoCollector;
+import com.bakdata.conquery.models.query.visitor.QueryVisitor;
+
+import lombok.Getter;
+import lombok.Setter;
+
+
+@Getter
+@Setter
+@CPSType(id = "ARRAY_QUERY", base = IQuery.class)
+public class ArrayQuery implements IQuery {
```

Comment:
Etwas Dokumentation

Reply:
Upps, hab die bisher nur in die PRs geschrieben